File: Programmers/pro_12901_2016년.py
def solution(a, b):
    day = ["SUN","MON","TUE","WED","THU","FRI","SAT"]
    month = [31,29,31,30,31,30,31,31,30,31,30,31]
    
    return day[(sum(month[:a-1])+b+4)%7]

File: Programmers/pro_12901_2016년.js
function solution(a, b) {
    const dayArr = ["THU", "FRI", "SAT", "SUN", "MON", "TUE", "WED"]
    const MonthArr = [0, 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    let daySum = b
    while (a--) {
        daySum += MonthArr[a]
    }
    return dayArr[daySum % 7];
}